source/index.html

```diff
@@ -23,6 +23,11 @@ <h4>Comments</h4>
     </ul>
 
     <button id="new_comment_button">New Comment</button>
+    <form id="new_comment" name="comment_form">
```

Comment:
Without editting the html can you think of a way to do this using javascript/jquery?